bots/tasks.py

```diff
@@ -211,6 +216,49 @@ def send_broadcast_msg(
         # save_broadcast_message(convo, text, msg_id)
 
 
+@shared_task
+def run_all_scheduled_functions():
+    for sf in BotIntegrationScheduledFunction.objects.select_related(
+        "bot_integration"
+    ).exclude(last_run_at__gt=timezone.now() - timedelta(hours=23)):
```

Comment:
@devxpy i had to make this `timedelta(hours=23)` instead of `days=1` because the tasks will be started after `H:00` on day 1, and we want to re-run those that ran after `H:00` and before `H+1:00`.